### DISCUSSION TITLE
Kata modulo 5

### DISCUSSION THREAD

**Discussion by erick129n** [LaunchX-InnovaccionVirtual/CursoIntroPython#142] [Q&A] [answered]
Hola comunidad, estaba realizando el ejercicio 2 de la kata 5, todo iba bien pero, al intentar hacer la resta me marca un error.¿ alguien me puede ayudar con el error?
![image](https://user-images.githubusercontent.com/95593792/153649376-42330948-327f-4f23-a583-f8021bbf7cfc.png)

**Comment by chistoperez** [comment, marked as answer]
Cuando recoges los datos del input son tipo string, osea texto, tienes que pasarlos a número para que se puedan restar. Intenta hacerlos enteros con int()

**Reply by erick129n** [reply]
si los tenia así. mi error fue no ejecutar el código int(). muchas gracias por la ayuda, ya funciono :D

**Comment by IdeIsaac** [comment]
Intenta poner abs(variable1-variable2) y ojo con querer concatenar distintos tipos de datos, quizás te salga el error por que intentas sumas caracteres con números, y por eso salga lo de str. Saludos
